### PR TITLE
add parser for bandi scanner

### PR DESCRIPTION
**Pull request by TalonsLee**
Signed-off-by: Xin Li <talons.lee@gmail.com>

### CODE REVIEW

**Review by Zhengchai — COMMENTED** [review]
this is a big change, can you add one readme to describing how to use this scanner? and its testing results etc. how is this one will help? Also looks the travis is failed.

**Comment by liulinC** [comment]
The PR looks good,  the commit add a parser to read the `bandit Scaner` result which is a json file and format the output.

The question about this PR is, is there any plain to integrate the `bandit scanner` into this repo, or it has already been integreted.

File: bandit_parser.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+#
+# Copyright (C) Citrix Systems Inc.
+#
+# This program is free software; you can redistribute it and/or modify
```
Comment (minli1) [inline bandit_parser.py:5]: As there already existing LICENSE file, so here may not be needed.

**Review by minli1 — APPROVED** [review]

**Comment by TalonsLee** [comment]
> The PR looks good, the commit add a parser to read the `bandit Scaner` result which is a json file and format the output.
> 
> The question about this PR is, is there any plain to integrate the `bandit scanner` into this repo, or it has already been integreted.

Bandit will not be integrate to this repo.
Bandit lastest docker image will be used in jenkins "security scan" stage,

**Comment by TalonsLee** [comment]
> this is a big change, can you add one readme to describing how to use this scanner? and its testing results etc. how is this one will help? Also looks the travis is failed.

This failure is not related to this commit.

**Comment by liulinC** [comment]
Merge this commit as it is a good enhancement and has no impact to the original code.

**Comment by Zhengchai** [comment]
do you have test result please? or a readme about how to use this tool?

**Comment by liulinC** [comment]
@TalonsLee Please intergrate the bandit scan into the Jenkins as well.

File: bandit_parser.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
```
Comment (robertbreker) [inline bandit_parser.py:5]: Hi @liulinC,

this license conflicts with /LICENSE.
Can this please either be
* changed to 2-clause-BSD
* or include the full GNU Lesser General Public License text in this repo AND specify in /LICENSE. that this file is under a different license than everything else.

Comment (liulinC) [reply bandit_parser.py:5]: thanks @robertbreker 
@TalonsLee, could you please help to fix it?

**Pull request merged**